core/drand_status.go

```diff
@@ -0,0 +1,93 @@
+package core
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/drand/drand/protobuf/drand"
+)
+
+const UnknownDesc = "Unknown"
+const NotStartedDesc = "Not started"
+const InProgressDesc = "In progress"
+
+type DkgStatus uint32
+
+const (
+	DkgReady DkgStatus = iota
+	DkgInProgress
+	DkgNotStarted
+)
+
+type ReshareStatus uint32
+
+const (
+	ReshareNotInProgress ReshareStatus = iota
+	ReshareInProgress
+)
+
+type BeaconStatus uint32
+
+const (
+	BeaconNotInited BeaconStatus = iota
+	BeaconInited
+)
+
+func GetDkgStatusDescription(value DkgStatus) string {
+	switch value {
+	case DkgReady:
+		return "Done"
+	case DkgInProgress:
+		return InProgressDesc
+	case DkgNotStarted:
+		return NotStartedDesc
+	default:
+		return UnknownDesc
+	}
+}
+
+func GetReshareStatusDescription(value ReshareStatus) string {
+	switch value {
+	case ReshareInProgress:
+		return InProgressDesc
+	case ReshareNotInProgress:
+		return NotStartedDesc
+	default:
+		return UnknownDesc
+	}
+}
+
+func GetBeaconDescription(value BeaconStatus) string {
+	switch value {
+	case BeaconNotInited:
+		return "Not inited"
+	case BeaconInited:
+		return "Inited"
+	default:
+		return UnknownDesc
+	}
+}
+
+func StatusResponseToString(status *drand.StatusResponse) string {
```

Comment:
i don't see these functions being used anywhere. they appear to be for use in testing / debugging. is that the intention?
Should they be directly in `core/` if so?

Reply:
This fn is not being used on this PR. I will be used on an upcoming PR. I can remove it and add it again on the next PR.

Reply:
do we need all of the functions here to be public on the core package?
can we put the diagnostic ones in a test package so that it isn't in the final binary?

Reply:
Removed.